nixos/tests/sylkserver.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  ...
+}:
+
+let
+  ports = {
+    sip = 5060;
+    xmpp = 5269;
+    web = 10888;
+  };
+in
+
+{
+  name = "sylkserver";
+  meta.maintainers = [ lib.teams.ngi.members ];
+
+  nodes.machine =
+    { config, ... }:
+    {
+      services.sylkserver.enable = true;
+
+      services.sylkserver.settings.config.SIP.local_ip = "0.0.0.0";
+      services.sylkserver.settings.config.SIP.local_tcp_port = ports.sip;
+
+      services.sylkserver.settings.xmppgateway.general.local_ip = "0.0.0.0";
+      services.sylkserver.settings.xmppgateway.general.local_port = ports.xmpp;
+
+      # TODO:
```

Comment:
There are some TODOs in this file, are they required to be solved before merging?